src/screens/ProfileScreen.js

```diff
@@ -215,9 +216,9 @@ export default class ProfileScreen extends React.Component {
     } else if (params.sent_friend_request || params.is_subscribed_to || this.state.sentConnectRequest) {
       return (
         <Button
-          style={margins.marginTop.md}
+          style={[margins.marginTop.md, {backgroundColor: colors.gray}]}
           onPress={connectMethod}
-          text="CONNECT"
+          text={params.is_subscribed_to ? "FOLLOWING" : "REQUESTED"}
```

Comment:
what if the user has not sent a request to this person yet?

Reply:
I think if you view the whole file, its under these lines